src/Fantomas.CoreGlobalTool.Tests/TestHelpers.fs

```diff
@@ -36,4 +48,11 @@ let private runFantomasTool arguments =
 
 let checkCode file =
     let arguments = sprintf "--check \"%s\"" file
-    runFantomasTool arguments
\ No newline at end of file
+    runFantomasTool arguments
+
+let formatCode file =
```

Comment:
Is there a reason for not just using `runFantomasTool`, instead of defining a new function `formatCode`?

Reply:
`runFantomasTool` is marked as `private`, so I exposed it with a new function.
Maybe a bit silly indeed.